docs/sensors.md

```diff
@@ -0,0 +1,216 @@
+          panels/street lighting, etc.) measure, monitor, and steer
+          energy sources.
+
+    -   Power suppliers use sensors to gather insights and offer incentives to users to use energy at non-peak times to reduce the costs of building greater capacities.
+
+#### Water sector
+
+-   **Wastewater and drinking water treatment plants:** sensors measure
+    water quality and flow to remotely monitor and steer treatment,
+    inlet- and outlet water quality, and support operations.
+
+-   **Water and wastewater networks:** sensors are used to monitor network conditions (flow metering and pressure sensors) and
+  steering (via valves, pumps).
+
+- **Meterological and hydrological networks**:
+
+    -   Sensors measure hydrological situations (precipitation, evaporation, temperature, river runoff, sunshine hours, etc.).
+
+    -   Sensors measure water quality and quantity (levels of groundwater and surface waters in reservoirs and water quality
+  parameters).
+
+-   Remote data transfer (mobile cards, internet), remote control, analysis of the meterological, and hydrological data in
+  computer-based systems.
+
+#### Mobility sector
+
+-   **Network capacity:** sensors monitor load in interconnected
+    charging stations for e-vehicles.
+
+-   **Traffic:** sensors can help in incident reporting and response, traffic directions, and parking management. They can also measure air pollution.
+
+#### Biodiversity/Environment sector
+
+-   **Forest and ocean protection:** sensors may be used to stop illegal
+    deforestation, protect endangered species, and track waste dumping.
+
+-   **Smart farming:** sensors measure methane levels, air temperature, rainfall, soil moisture, soil pH, etc.
+
+### Step 3: What Do I Need to Consider When Acquiring Sen- sors?
+
+  There are many platforms available on/offline for purchase-by- unit.
+  Users may choose sensors based on:
+ 
+  **size** (nano, micro, 2D, 3D, etc.)
+ 
+ type of **connection**: wired/wireless
+ 
+  **functional** requirements (offline usage and range, solarpower,
+  calibration resolution, repeatability, interferences, environmen- tal
+  conditions, maintenance, etc.)
+ 
+  **non-functional** attributes (e.g., longevity, interoperability, scalability, wireless protocols, compliance with safety standards,
+  etc.)
+ 
+  **costs** (can be mitigated by industry collaboration)
+ 
+  **accuracy** as precision requirements often drive up costs
+  exponentially. Sensors often require frequent calibration to assure
+  correct measurements, so you need to check if calibration services are
+  available locally.
+ 
+  **disponsability.** Sensors may offer many advantages: low-cost, easy-to-use, short-term or rapid single-point measurements, reliable
+  information and digital connectivity for availability to users and
+  centralized /decentralized facilities.
+
+### Procurement of Sensor Data
+Sensor datasets are siloed, often proprietary, and treated as
```

Comment:
this paragraph can be formatted just as line 9 - 20. Please have some spacing inserted for the whole paragraph